src/Microsoft.AspNetCore.Hosting/WebHostExtensions.cs

```diff
@@ -15,11 +16,61 @@ namespace Microsoft.AspNetCore.Hosting
     public static class WebHostExtensions
     {
         /// <summary>
+        /// Starts the host.
+        /// </summary>
+        /// <param name="host"></param>
+        /// <returns></returns>
+        public static void Start(this IWebHost host)
+        {
+            host.StartAsync().GetAwaiter().GetResult();
+        }
+
+        /// <summary>
+        /// Starts the host.
+        /// </summary>
+        /// <param name="host"></param>
+        /// <returns></returns>
+        public static Task StartAsync(this IWebHost host)
+        {
+            return host.StartAsync(CancellationToken.None);
+        }
+
+        /// <summary>
+        /// Gracefully stops the host.
+        /// </summary>
+        /// <param name="host"></param>
+        /// <returns></returns>
+        public static Task StopAsync(this IWebHost host)
+        {
+            return host.StopAsync(CancellationToken.None);
```

Comment:
A little, but there wasn't a way to expose the timeout to the extensions.  Really CancellationToken.None means you're not going to be cancelled externally, it doesn't prevent you from cancelling internally. HttpClient does something like this actually, it always chains its own token to yours with the default timeout.